_posts/2020-10-19-Software-Synth.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Software Synth in Jamulus"
+lang: "en"
+author: "niebert"
+heading: "Software Synth in Jamulus"
+---
+This post will discuss how to setup a software synth in Jamulus.
+<!--more-->
+
+## Linux
+We will elaborate the concept of playing a software synth on Linux as example and transfer that to other operating systems (e.g. Windows, MacOSX).
+* **(Hammond Organ Synth)** As an example we talk open source software synth [setBfree](https://github.com/pantherb/setBfree), which is a hammond organ emulator that outputs through JACK on Linux.
```

Comment:
The beginning of this sentence reads wrong.
Probably it should be something around "We're going to talk about the open source software synth[...]".